packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -42,28 +42,37 @@ abstract class EndpointDispatch {
       var module = modules[modulePackage];
       if (module == null) return null;
 
-      connector = module.connectors[endpointName];
+      connector = module.connectors[_endpointFromName(endpointName)];
       if (connector == null) return null;
     }
 
     return connector;
   }
 
+  String _endpointFromName(String name) {
+    var components = name.split('/');
+    return components[0];
+  }
+
   /// Dispatches a call to the [Server] to the correct [Endpoint] method. If
   /// successful, it returns the object from the method. If unsuccessful it will
   /// return a [Result] object.
-  Future<Result> handleUriCall(Server server, String endpointName, Uri uri,
-      String body, HttpRequest request) async {
-    var endpointComponents = endpointName.split('.');
+  Future<Result> handleUriCall(
+    Server server,
+    String path,
+    Uri uri,
+    String body,
+    HttpRequest request,
+  ) async {
+    var endpointComponents = path.split('.');
     if (endpointComponents.isEmpty || endpointComponents.length > 2) {
-      return ResultInvalidParams(
-          'Endpoint $endpointName is not a valid endpoint name');
+      return ResultInvalidParams('Endpoint $path is not a valid endpoint name');
     }
```

Comment:
Question: Is this only created for the check here? And if so, is the check required? Can't see that `endpointComponents` is used in this scope in any other way.